src/plugins/FileService/utils/file.ts

```diff
@@ -0,0 +1,28 @@
+export function toUint8Array(blob: Blob) {
```

Comment:
Use `Blob.prototype.toArrayBuffer`, and add polyfill for Safari.

Reply:
your adde the polyfill to master branch?
i merge master